src/core/Buttons/DropdownButton/DropdownButton.tsx

```diff
@@ -19,7 +19,13 @@ export type DropdownButtonProps = {
    * and returns a list of `MenuItem` components.
    */
   menuItems: (close: () => void) => JSX.Element[];
-} & Omit<ButtonProps, 'onClick' | 'styleType' | 'endIcon'>;
+  /**
+   * Style of the dropdown button.
+   * Use 'borderless' to hide outline.
+   * @default 'default'
+   */
+  styleType?: 'default' | 'borderless';
```

Comment:
When switching between style types menu with stays the same:
![image](https://user-images.githubusercontent.com/36186912/115503950-df8c6b00-a27f-11eb-85f0-90896669fcfb.png)
It's a minor thing. Strange but when changing other props in Storybook, menu width changes accordingly.

Reply:
Oh, I found why 😀
![image](https://user-images.githubusercontent.com/36186912/115504402-8ffa6f00-a280-11eb-8304-6f50e83caf44.png)
We need to add `styleType` to deps list.

Reply:
Yeah I noticed that, but is it really needed? Who is gonna change styleType in practice?

Reply:
But it is an easy fix and since we know that it will bug me 😀

Reply:
Yeah that's why I already did it 😄